src/styles/logo.scss

```diff
@@ -0,0 +1,4 @@
+.logo {
+  margin-left: 50px;
```

Comment:
It's better to make logo as an element of the header

Reply:
At first I done this way, but I got recommended on Q&A to make it a separate element, because it can be used not only in header.